docs/index.rst

```diff
@@ -395,6 +395,42 @@ Example using JSON format:
     not validate them. If an unexpected value is found, the command will fail
     with a stack trace showing the error.
 
+xml-test-run command
+++++++++++++++++++++
+
+The xml-test-run command generates an XML file suited to be imported by the
+Test Run XML importer. It takes a valid xUnit XML file and a Python test suite
+where test case IDs can be found and generates a resulting XML file with all
+the information the Test Run XML importer needs.
+
+The xml-test-run command only requires you to pass the path to the xUnit XML
+file, the path to the Python test suite source code, the Polarion user ID, the
+Polarion project ID and the output XML file path (it will override if the file
+already exists).
+
+Even though ``--response-property`` is optional, it is highly recommended to
+pass it because will be easier to monitor the importer messages (which is not
+handled by Betelgeuse).
+
+The example below shows how to run xml-test-run command:
+
+.. code-block:: console
+
+    $ betelgeuse xml-test-run \
+        --response-property property_key=property_value \
+        sample_project/results/sample-junit-result.xml \
+        sample_project/tests/ \
+        testuser \
+        PROJECT \
+        output.xml
+
+The xml-test-run command can set test run custom fields.  The
+``--custom-fields`` option can be used with a ``key=value`` format or a JSON
+format as explained in `test-run command`_ section. Make sure to pass the the
+custom field ID (same as in Polarion) and its value. Make sure to pass custom
+field values as string since they will be converted to XML where there is no
+type information.
+
```

Comment:
Move these two statements starting with `Make sure to pass` to a `warning` section